addons/fabric8/fabric8.addon

```diff
@@ -0,0 +1,40 @@
+# Name: fabric8
+# Description: Installs fabric8 locally on Minishift
+
+# TODO enable CORS!
+
+oc login -u system:admin
```

Comment:
ah great, thanks for the heads up. You learn something new about `oc` every day it seems ;)